content/en/docs/Writing policies/generate.md

```diff
@@ -246,7 +246,111 @@ spec:
           foo: bar
 ```
 
-## Generating resources into existing namespaces
+## Generate for Existing resources
+
+With Kyverno 1.7.0+, Kyverno supports the generate existing resources. Generate existing policies are applied in the background which creates target resource in existing matched resources in the cluster.
+They may also optionally be configured to apply upon updates to the policy itself.
+
+{{% alert title="Note" color="warning" %}}
+The proper permissions may need to be granted to Kyverno ServiceAccount. Kubernetes resources may require granting additional privileges to the ClusterRole responsible for the generate behavior. To enable Kyverno to generate these other types, edit the ClusterRole typically named kyverno:generate and add or update the rules to cover the resources and verbs needed.
+{{% /alert %}}
+
+### Generate NetworkPolicy on Existing Namespaces
```

Comment:
Do we want to use the sub-title for example policies?

Reply:
With sub-title its looks more clear and easy to understand, WDYT ?

Reply:
I'm fine with this.

Reply:
Also we have 2 different policies , so having with sub-titles helps